torchtune/training/_distributed.py

```diff
@@ -22,46 +22,16 @@
 )
 from torch.distributed.checkpoint.state_dict import _init_optim_state
 from torch.distributed.fsdp import ShardingStrategy
-from torch.distributed.fsdp.wrap import ModuleWrapPolicy
 from torch.optim import Optimizer
 from torchao.dtypes.nf4tensor import NF4Tensor, to_nf4
 from torchtune.modules import TransformerDecoder
-from torchtune.modules.peft import DoRALinear, LoRALinear
-from torchtune.modules.peft.lora import _lora_a_init_params, _lora_b_init_params
 from torchtune.utils import get_logger
 
 from torchtune.utils._device import get_device
 
 _log: logging.Logger = get_logger()
 
 
-FSDPPolicyType: Type = Callable[[nn.Module, bool, int], bool]
```

Comment:
I think you need to remove the reference to this in `api_ref_training.rst` for the docs to build.